study-builder/studies/pancan/study-events.conf

```diff
@@ -119,10 +119,8 @@
 //        "type": "SENDGRID_EMAIL",
 //        "templates": [
 //          {
-//            "emailTemplate": ${emails.joinMailingList},
-//            "language": "en",
-//            "isDynamic": true
-//          }
+//            "emailTemplate": ${emails.en_consent_completed}, "language": "en", "isDynamic": true },
```

Comment:
Not sure if I applied the correct event.
It is not quite clear from [the DDP-6358 ticket](https://broadinstitute.atlassian.net/browse/DDP-6358).